### DISCUSSION TITLE
specifying residuals points to solve pde

### DISCUSSION THREAD

**Discussion by rim-2023** [lululxvi/deepxde#1152] [Q&A]
hiii i have a problem i want to solve a pde with pinns and i use deepxde but i want to choose my residuals points how? thx

**Comment by lululxvi** [comment]
See examples at https://deepxde.readthedocs.io/en/latest/demos/pinn_forward.html

**Comment by rim-2023** [comment]
hi how to plot only the function cost according to numbers of iterations by choosing the writing on the two axes? 
i try this code : saveplot(losshistory, train_state, issave=True, isplot=True)  but in this code it displays on the figure test metrics too in addition I have no right to change the writing on the figure

**Reply by lululxvi** [reply]
I cannot understand your question.
